spec/features/items/user_can_view_all_items_spec.rb

```diff
@@ -2,14 +2,60 @@
 
 describe "When user visits root" do
   scenario "They see list of all items" do
-    item = Fabricate(:item)
-    item2 = Fabricate(:item)
-
+    item, item2 = Fabricate.times(2, :item)
     visit root_path
 
     expect(page).to have_content(item.title)
     expect(page).to have_content(item.price)
     expect(page).to have_content(item2.title)
     expect(page).to have_content(item2.price)
   end
+
+  scenario "They see a list of most popular items" do
+    item1, item2, item3, item4 = Fabricate.times(4, :item)
+    unordered_items = Fabricate.times(10, :item)
+    user1 = Fabricate(:user)
+
+    order1_total = (item1.price * 2) + item3.price + item2.price
+    order1 = user1.orders.create!(total_price: order1_total)
+    OrderItem.create!(order: order1,
+                      item: item1,
+                      quantity: 2,
+                      price: item1.price)
+    OrderItem.create!(order: order1,
+                      item: item2,
+                      quantity: 1,
+                      price: item2.price)
+    OrderItem.create!(order: order1,
+                      item: item3,
+                      quantity: 1,
+                      price: item3.price)
+
+    order2_total = (item3.price * 4) + item4.price
+    order2 = user1.orders.create!(total_price: order2_total)
+    OrderItem.create!(order: order2,
+                      item: item3,
+                      quantity: 4,
+                      price: item3.price)
+    OrderItem.create!(order: order2,
+                      item: item4,
+                      quantity: 1,
+                      price: item4.price)
+    visit root_path
+
```

Comment:
@meyerhoferc @valerietrudell @CjMoore thoughts on adding an OrderItem fabricator? Happy to take that on to get some more experience in fabricator.